src/components/tabs/tabs.component.tsx

```diff
@@ -182,6 +182,8 @@ const Tabs = ({
 
   /** Handles the changing of tabs with the mouse */
   const handleTabClick = (ev: React.MouseEvent<HTMLElement>) => {
+    // istanbul ignore if
+    // (code doesn't seem to be ever reached, the keydown event on TabTitle calls stopPropagation)
```

Comment:
**question:** if this line of code is unreachable, should we open a ticket to investigate if it's worth removing it?

Reply:
now done, thanks (I won't link to Jira here but I've put the ticket number in the comment in my code changes). I also realised the explanation I'd given in this comment of why it's apparently unreachable wasn't correct, the ticket has a more detailed description of why (in my opinion) it can't be reached.